src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1446,6 +1446,9 @@ enum Mode {
     """TRAM"""
     TRAM
 
+    """Sharing a your own private car with others as passengers."""
+    CARPOOL
```

Comment:
As a user of OTP, it would be more: car pool as passenger into someone else's car.

Reply:
I've changed it to a generic "Sharing a private car with others.". How do you like that?

Reply:
I've used "Private car trips shared _with_ others." which seems more correct in English to me but I'm happy if a native speaker could cross-check that.